### ISSUE TITLE
Install fails because nginx refuses proxy_max_temp_file_size value

### ISSUE THREAD

**Issue by alexAubin** [YunoHost-Apps/codimd_ynh#38]
**Describe the bug**

Today a user reported that the install failed with : 

```
nginx: [emerg] "proxy_max_temp_file_size" directive invalid value in /etc/nginx/conf.d/pad.maindomain.tld.d/codimd.conf:21
```

Corresponding line in nginx.conf : https://github.com/YunoHost-Apps/codimd_ynh/blob/master/conf/nginx.conf#L21

Apparently commenting the line seems to solve the issue ... I'm puzzled on why these lines are necessary (8192m sounds like **a lot** ?!)

**Versions**
- Hardware: No idea / this is from another user
- YunoHost version: 3.8.5.4
- I have access to my server: No idea / this is from another user
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance ?: No idea / this is from another user

**To Reproduce**

Try to install the app ... though not sure where / how to reproduce exactly considering the tests on the CI are passing and so far no other users complained about it

**Expected behavior**

Instal completing successfully

**Logs**

https://paste.yunohost.org/raw/nilixeyati

**Comment by ericgaspar** [comment]
Those values ​​were found (　･ω･)☞ [here](https://hackmd.io/c/codimd-documentation/%2F%40codimd%2Fweb-server-nginx)

#39  lowers the values to  :
```
client_max_body_size 256m;              # Nginx as 1M by default
proxy_max_temp_file_size 1024m;         # Nginx as 1024M by default
```

**Comment by ericgaspar** [comment]
Fixed by #39
